# see available mappings on src/bender_joy/xbox.py

# pause button
b_pause: 'START'

# neck keys
b_neck_actuate: 'RS'
a_neck_sides: 'RS_HORZ'
a_neck_front: 'RS_VERT'

# emotion buttons
b_increment: 'LB'
b_decrement: 'RB'
b_happy: 'A'
b_angry: 'B'
b_sad: 'X'
b_surprise: 'Y'

max_neck_degrees: 60
max_emotion_intensity: 3
min_emotion_intensity: 1